docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -3,12 +3,148 @@ layout: docs
 title: Test Code Structure
 permalink: /documentation/test-api/test-code-structure.html
 ---
-# Stub
+# Test Code Structure
 
-## fixtures
+This topic contains the following sections:
 
-## specifying the start webpage
+* [Fixtures](#fixtures)
+  * [Specifying the Start Webpage](#specifying-the-start-webpage)
+* [Tests](#tests)
+  * [Test Controller](#test-controller)
+* [Initialization and Clean-Up](#initialization-and-clean-up)
 
-## tests
+## Fixtures
 
-## test-controller
\ No newline at end of file
+TestCafe tests must be organized into categories called *fixtures*.
+A JavaScript file with TestCafe tests can contain one or more fixtures.
+
+To declare a test fixture, use the `fixture` function.
+
+```text
+fixture( fixtureName )
+```
+
+```text
+fixture `fixtureName`
+```
+
+Parameter     | Type   | Description
+------------- | ------ | ------------------------
+`fixtureName` | String | The name of the fixture.
+
+[Tests](#tests) that constitute a fixture go after this declaration.
+
+### Specifying the Start Webpage
+
+You can optionally use the fixture declaration to specify a webpage at which all tests in a fixture start.
+To do this, use the `page` function.
+
+```text
+page( url )
+```
+
+Parameter | Type   | Description
+--------- | ------ | ------------------------------------------------
+`url`     | String | The URL of the webpage at which the tests start.
+
+```js
+fixture `My fixture`
+    .page('http://example.com');
+```
+
+If the start page is not specified, it defaults to `about:blank`.
+
+> You can also call the [Navigate action](actions/navigate.md) at the beginning of a test
+> to navigate to the start page.
+
+## Tests
+
+To introduce a test, call the `test` function and pass test code inside.
+
+```text
+test( testName, func(t) )
+```
+
+Parameter  | Type     | Description
+---------- | -------- | --------------------------------------------------------------------
+`testName` | String   | The test name. Must be unique within a fixture.
+`func`     | Function | An asynchronous function that contains test code.
```

Comment:
When I dealt with .NET only, I would definitely choose `function` as a parameter name (actually, no, because I would name it meaningfully). But the more I work with js, the more I get used to contractions like `func` as parameter names.